stdlib/contrib/RohanSreerama5/naiveBayesClassifier/README.md

```diff
@@ -0,0 +1,65 @@
+# Naive-Bayes-Classifier-Flux
+Implementation of Naive Bayes Classifier in Flux
+
+Awesome! So you want to get started with ML in Flux? Let's dive in! 
+
+Steps: 
+
+Prerequesites: InfluxDB 2.0 (local or Cloud instance), this repo (includes zoo-data.csv, script.py and naiveBayesClassifier.flux) 
+InfluxDB: https://github.com/influxdata/influxdb
+
+1) Ensure you have InfluxDB 2.0 set up by either going to localhost:9999 (if you `make` InfluxDB from source) or going to your Cloud instance on AWS, Azure, etc. Keep the instance running throughout this demo.  
+2) Choose a binary dataset (fields can only take on 2 unique values) or visit https://archive.ics.uci.edu/ml/datasets.php for some wonderful datasets you can get started with. 
+
+![](images/csvData.png)
```

Comment:
Nathaniel pointed out that we don't typically added images to READMEs with data that can be pasted in as text. Would you mind removing the images and adding this data as test or linking to the csvs in this project?